src-bs/CCChar.ml

```diff
@@ -0,0 +1,24 @@
+(* This file is free software, part of containers. See file "license" for more details. *)
+
+(** {1 Utils around char}
+
+    @since 0.14 *)
+
+include Char
+
+let equal (a:char) b = Pervasives.(=) a b
+
+let pp_buf = Buffer.add_char
+let pp = Format.pp_print_char
```

Comment:
not sure if BS likes this, I think they disliked that containers defines a lot of printers because their compiler doesn't like the big Format GADT?

Reply:
Ah interesting - it seems to compile ok now, perhaps they sorted that issue?

Reply:
aha it might be a runtime 'not implemented' kind of thing - just hit a few of these.